kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/array.kt

```diff
@@ -0,0 +1,95 @@
+import io.kotest.assertions.json.pretty
```

Comment:
I think this is missing a package statement, isn't it?

Reply:
Yup, you're right. Will fix :)